src/sync.cpp

```diff
@@ -78,12 +78,12 @@ namespace librealsense
         :matcher({stream})
     {
         _streams_type = {stream_type};
-        _name = "I " + std::string(rs2_stream_to_string(stream_type));
+        _name = std::string(rs2_stream_to_string(stream_type));
     }
 
     void identity_matcher::dispatch(frame_holder f, const syncronization_environment& env)
     {
-        LOG_IF_ENABLE(_name << "--> " << *f.frame, env);
+        //LOG_IF_ENABLE( "--> identity_matcher: " << _name, env );
```

Comment:
why removed?

Reply:
It didn't add anything

Reply:
It's important to track all the way the frame did , this is the only way to debug it.